package.json

```diff
@@ -95,25 +95,28 @@
 		"files": [
 			{
 				"path": "./js/build/*.js",
-				"maxSize": "2 kB"
+				"maxSize": "3 kB"
 			},
 			{
 				"path": "./js/build/index.js",
-				"maxSize": "1 mB"
+				"maxSize": "3 mB"
 			},
 			{
 				"path": "./js/build/*.css",
 				"maxSize": "1 kB"
 			},
 			{
 				"path": "./js/build/index.css",
-				"maxSize": "8 kB"
+				"maxSize": "10 kB"
 			},
 			{
 				"path": "./google-listings-and-ads.zip",
 				"maxSize": "12 mB",
 				"compression": "none"
 			}
-		]
+		],
+        "ci": {
+            "trackBranches": ["trunk", "develop"]
+        }
```

Comment:
@tomalec , just noticed this, really minor nitpick: the three lines here are using space indentation instead of tabs like other lines. 😄

Reply:
I'd address it in https://github.com/woocommerce/google-listings-and-ads/pull/869